doc/configuration.rst

```diff
@@ -4,7 +4,7 @@ Configuration
 All configuration can be done by adding a configuration files. They are looked for in:
 
  * ``/etc/luigi/client.cfg``
```

Comment:
@soxofaan Did you deliberately not change the documentation here? Because in the code you wrote that client.cfg is deprecated.